lib/membrane_flac_plugin/membrane_flac_parser.ex

```diff
@@ -65,45 +96,151 @@ defmodule Membrane.FLAC.Parser do
               {:stream_format, {:output, format}}
 
             %Buffer{} = buf ->
-              {:buffer, {:output, set_buffer_pts(buf, state)}}
+              {:buffer, {:output, buf}}
           end)
 
+        {actions, state} = calculate_pts(actions, state)
```

Comment:
maybe `calculate_pts` -> `calculate_buffers_pts`, since this function calculates `pts` values of buffers contained in the actions list passed as the first argument?